templates/patient/card/rx.html.twig

```diff
@@ -1,5 +1,15 @@
 {% extends "patient/card/card_base.html.twig" %}
 
 {% block content %}
+
+    {# Display the Edit button for openemr prescriptions#}
+    {% if weno == true %}
+        <div class="card-title mb-0 d-flex p-1 justify-content-between">
+            <span class="font-weight-bolder text-primary">Prescription History</span>
```

Comment:
translate (prob also need to do this in master)

Reply:
fixed this with another WENO PR will push soon